apache/zeppelin
# [ZEPPELIN-2544] clean task doesn't work in zeppelin-web

### What is this PR for?

`prebuild` task in package.json doesn't work for `build:*` tasks in zeppelin-web

otherwise we will get overwritten result, if previous build for zeppelin-web failed like

```
[INFO] Running "cacheBust:taskName" (cacheBust) task
[INFO] Warning: Unable to write "/Users/jl/local/src/g/zeppelin/zeppelin-web/dist/styles/vendor.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.e7906afb60fa5267.css" file (Error code: ENAMETOOLONG). Use --force to continue.
```

### What type of PR is it?
[Bug Fix]

### Todos

DONE

### What is the Jira issue?

[ZEPPELIN-2544](https://issues.apache.org/jira/browse/ZEPPELIN-2544)

### How should this be tested?

- `cd zeppelin-web`
- `npm run build:dist`

Then, check the console output whether `prebuild` is applied or not.

### Screenshots (if appropriate)

```
## Before

$ zeppelin-web git:(ZEPPELIN-2544/prebuild-does-not-work) npm run build:dist

> zeppelin-web@0.0.0 build:dist /Users/1ambda/github/apache/apache-zeppelin/zeppelin-master/zeppelin-web
> grunt pre-webpack-dist && webpack && grunt post-webpack-dist

## After

zeppelin-web git:(ZEPPELIN-2544/prebuild-does-not-work) ✗ npm run build:dist

> zeppelin-web@0.0.0 build:dist /Users/1ambda/github/apache/apache-zeppelin/zeppelin-master/zeppelin-web
> npm-run-all prebuild && grunt pre-webpack-dist && webpack && grunt post-webpack-dist
```

### Questions:
* Does the licenses files need update? - NO
* Is there breaking changes for older versions? - NO
* Does this needs documentation? - NO
